docs/userManual.md

```diff
@@ -465,9 +465,9 @@ The following menu allows the user to specify the data to load.
   to, which is Cassandra. We also support another DB, 'mysql' and 
   'impala'.
 
-* host: One of the network name/IP address of the machines hosting 
-  Cassandra cluster. Every node in Cassandra cluster are the same 
-  (no master-slave concept), thus Hillview can connect to any node.
+* host: The network name/IP address of Hillview's root node. A root 
```

Comment:
are you sure about this?
what is this address used for?
if this is correct you have to also say in the deployment section that the root node has to be colocated with a cassandra node.

Reply:
Sorry, my mistake. I just did more testing and now Hillview can connect to any cassandra node. The original description should be fine. I will revert the file.